common/sgx/tcbinfo.h

```diff
@@ -36,6 +36,56 @@ typedef union _oe_tcb_level_status {
 
 } oe_tcb_level_status_t;
 
+typedef enum _oe_tcb_status
```

Comment:
This enum has to be defined in a header file that's published (e.g. evidence.h). Otherwise, the claim values are meaningless to the caller. I'd also suggest we add value to each item in the enum. E.g.

```
OE_TCB_STATUS_SGX_UP_TO_DATE = 0,
OE_TCB_STATUS_SGX_OUT_OF_DATE = 1,
...
```

Reply:
Published the structures and update the Doxygen headers.